rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/PatchLogBase.java

```diff
@@ -175,12 +174,8 @@ public Version append(RDFPatch patch) {
         });
     }
 
```

Comment:
The following code was never called anywhere, so I removed it since it's bad practice to keep dead code around.